docker-compose-generator/docker-fragments/traefik-labels.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  btcpayserver:
+    labels:
+      - "traefik.backend=btcpayserver"
+      - "traefik.backend.loadbalancer.sticky=true"
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host:${BTCPAY_HOST}"
+      - "traefik.port.rule=49392"
+      - "traefik.acme.domains=${BTCPAY_HOST},www.${BTCPAY_HOST}"
+      - "traefik.acme.email=${LETSENCRYPT_EMAIL}"
```

Comment:
Why all this settings? It should be enough to just specify this:
- "traefik.enable=true"
- "traefik.frontend.rule=Host:${BTCPAY_HOST}"
As I know others are optional or can be set in traefik.toml.

Reply:
This is from 2018 lol. We'll most likely remove traefik support since it's not feasible to maintain 2 separate proxy options for minimal benefits